frontend/src/components/Board/OrderPCBs.js

```diff
@@ -12,8 +12,7 @@ const OrderPCBs = ({ zipUrl, boardSpecs }) => {
   const jlcpcbUrl = `https://cart.jlcpcb.com/quote?fileurl=${zipUrl}&from=kitspace`
 
   // TODO
-  // eslint-disable-next-line no-unused-vars
-  const trackClick = vendor => e => {
+  const trackClick = () => () => {
     // window.plausible('Order PCBs', {
     //   props: { project: this.props.project, vendor },
     // })
```

Comment:
Would undo getting rid of `vendor` here too, made more sense before. By the way, I think we should change this to just check if `window.plausible` exists all the time, and no run it if it doesn't.